packages/framework/ai-collab/README.md

```diff
@@ -238,10 +239,166 @@ To see detailed information about each event, please read their cooresponding [t
+
+Let's take a look at some examples for the following SharedTree application schema
+
+```ts
+import { aiCollab, DebugEvent, Diff } from "@fluidframework/ai-collab/alpha";
+
+const sf = new SchemaFactory("testApp");
+
+class Todo extends sf.object("Todo", {
+	id: sf.identifier,
+	title: string
+	description: string
+}) {}
+
+class TestAppRootObject extends sf.object("TestAppRootObject", {
+	id: sf.identifier,
+	todos: sf.array([Todo]),
+	innerObject: sf.object("InnerObject", {
+		nestedTodos: sf.array([Todo]),
+	}),
+}) {}
+
+const response = aiCollab({
+	openAI: {
+		client: new OpenAI({
+			apiKey: OPENAI_API_KEY,
+		}),
+		modelName: "gpt-4o",
+	},
+	treeNode: view.root,
+	prompt: {
+		systemRoleContext:
+			"You are a manager that is helping out with a project management tool. You have been asked to edit a group of tasks.",
+		userAsk: userAsk,
+	},
+	limiters: {
+		maxModelCalls: 25
+	}
+	planningStep: true,
+	finalReviewStep: true,
+	debugEventLogHandler: (event: DebugEvent) => {console.log(event);}
+});
+
+const Diffs: diff[] = response.Diffs
+```
+
+Each Diff will contain one or more `NodePath`'s. Each `NodePath` provides an array of objects that detail the path from the root node passed to ai-collab, down to the node targeted for editing. The first index in the `NodePath` is an object pointing to the target node and the last index is always the root node.
+
+Lets look at an example of the Insert Diff
+The following `InsertDiff` is an example of a `Diff` that would result from if the ai agent inserts an object into index 1 of `TestAppRootObject.rootVectors`
+### Example Insert Ui Diff
+```json
+type: "insert",
+nodePath: [
+		{
+			shortId: -14,
+			schemaIdentifier: "testApp.Todo",
+			parentField: 1,
+		},
+		{
+			shortId: undefined,
+			schemaIdentifier: "testApp.Array<[\"testApp.Todo\"]>",
+			parentField: "todos",
+		},
+		{
+			shortId: -1,
+			schemaIdentifier: "testApp.TestAppRootObject",
+			parentField: "rootFieldKey",
+		},
+	],
+  	aiExplanation: "I need to insert a todo within the todos array",
+  	nodeContent: {
+		id: "f75951b0-df9d-4daa-aaf2-c322f2f462a8",
+		title: "Example Todo Title",
+		description: "Example Todo Description"
+	},
+```
+As you can see, the object at the beginning of the `nodePath` array directly points to the newly inserted node while each next object is the parent of the preceding node until you hit the root node passed to the ai-collab function call.
+
+Using the following ui diff, you can identify the modified node in a number of different ways.
+
+The simplest way is to use the `shortId` where you can use the following code to identify the newly inserted node within the SharedTree.
+
+**Note that the shortId field will only exist for objects that have a field defined as the `SchemaFactory.identifier` field.** See the above example app schema in this section to see the schema field defined as `sf.identifier`
```

Comment:
You'll want to test this before merging, but GitHub supports a "notice" syntax for things like Notes and Warnings. I believe you should be able to do:
```suggestion
> [!Note]
> The `shortId` field will only exist for objects that have a field defined as the `SchemaFactory.identifier` field.

See the above example app schema in this section to see the schema field defined as `sf.identifier`
```